book/coming_from_cmd.md

```diff
@@ -0,0 +1,57 @@
+# Coming from CMD.EXE
+
+This table was last updated for Nu 0.66.0.
+
+| CMD.EXE                              | Nu                                               | Task                                                              |
+| ------------------------------------ | ------------------------------------------------ | ----------------------------------------------------------------- |
+| `ASSOC`                              |                                                  | Displays or modifies file extension associations                  |
+| `BREAK`                              |                                                  | Trigger debugger breakpoint                                       |
```

Comment:
Some of these should probably be cleaned up a bit. break doesn't trigger a break point in cmd.exe. https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/break

Reply:
According to `/?`, it does with extensions. Without extensions, it doesn't do anything under Windows.

Reply:
From that page,

No effect:

> Because the break command has no effect

Trigger breakpoint:

> If command extensions are enabled and running on the Windows platform, inserting the break command into a batch file enters a hard-coded breakpoint if being debugged by a debugger.

Reply:
oh, I see. you're right. I stopped reading after this part.

> Sets or clears extended CTRL+C checking on MS-DOS systems. If used without parameters, break displays the existing setting value.